src/apps/pillarx-app/components/TokensVerticalTile/TokensVerticalTile.tsx

```diff
@@ -41,9 +44,12 @@ if (!dataLeft || !dataRight || isDataLoading) {
 }
 
     return (
-    <TileContainer className='px-10 pt-[30px] pb-5 tablet:p-5 mobile:p-0 mobile:bg-[#1F1D23] mobile:flex-col'>
-        <TokensVerticalList position='left' data={dataLeft} title={titleLeft} />
-        <TokensVerticalList position='right' data={dataRight} title={titleRight} />
+    <TileContainer className='flex-col px-10 pt-[30px] pb-5 tablet:p-5 mobile:p-0 mobile:bg-[#1F1D23]'>
+        {meta?.display.title && <Body className='text-purple_light'>{meta.display.title}</Body>}
```

Comment:
Please optional chain everything here in meta - there is always a possibility the API doesn't give a display object / title, for example